plugins/sinks/columbus/meta.yaml

```diff
@@ -0,0 +1,12 @@
+name: "Columbus HTTP"
+version: "1.0.0"
+sample_config: |-
+  sinks:
```

Comment:
Sinks keyword we should avoid from sample config I think. as we would need to combine sample configs from multiple sinks when people add more than one.

Reply:
yeah, forgot to remove that, we should just put the config map.

@GrayFlash need help to remove that in the next push 🙏 .